kitsune/products/views.py

```diff
@@ -82,6 +83,8 @@ def document_listing(request, product_slug, topic_slug, subtopic_slug=None):
 
     documents, fallback_documents = documents_for(**doc_kw)
 
+    banner = cta_banner(products=[product], topics=[topic], page_type="topic", request=request)
```

Comment:
nit: I would keep the cta_banner in the context as it's easier to understand what kind of a banner this is